### DISCUSSION TITLE
[v9 Node] Installation does not support plain configs or .env

### DISCUSSION THREAD

**Discussion by Oreolek** [directus/directus#3395] [Archive]
CLI installation is abysmal. If I make a typo I have to start the configuration all over again. Can't you add a text config like before? Or at least mention that in the docs.

**Comment by rijkvanzanten** [comment]
There's a text config like before, that CLI tool generates it. You can run `npx directus bootstrap` instead of init to install the DB from the `.env` file

> CLI installation is abysmal.

If you have any suggestions on how to improve it, then please open a feature request / pr. Complaining without any feedback doesn't solve anything for anyone.

**Comment by Oreolek** [comment]
```
> npx directus bootstrap
✨ Initializing bootstrap...
Error: knex: Required configuration option 'client' is missing.
at new Client (node_modules/knex/lib/client.js:54:11)
```

Where's the `.env` sample?

**Reply by gensmann** [reply]
https://github.com/directus/directus/blob/main/api/example.env

https://docs.directus.io/reference/environment-variables.html

**Reply by Oreolek** [reply]
Well here's my problem: it can't connect to the database.

I wasn't sure in the CLI tool because I might paste a wrong password, or type a wrong password, it doesn't show what I'm typing so there's no way to check. But now with `.env` file I'm sure the connection parameters are right.

Node v15.4.0, Mariadb 10.3.27
